Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/TextLogFileAcceptanceTest.cs

```diff
@@ -289,12 +289,12 @@ public void TestOpenBeforeCreate()
 				new Action(() => logFile = new TextLogFile(_scheduler, fileName)).ShouldNotThrow();
 
 				logFile.Property(x => x.EndOfSourceReached).ShouldEventually().BeTrue(TimeSpan.FromSeconds(5));
-				logFile.Property(x => x.GetValue(LogFileProperties.EmptyReason)).ShouldEventually().Be(ErrorFlags.SourceDoesNotExist, TimeSpan.FromSeconds(5),
+				logFile.Property(x => x.Error).ShouldEventually().Be(ErrorFlags.SourceDoesNotExist, TimeSpan.FromSeconds(5),
 				                                                           "Because the specified file doesn't exist");
 
 				File.WriteAllText(fileName, "Hello World!");
 
-				logFile.Property(x => x.GetValue(LogFileProperties.EmptyReason)).ShouldEventually().Be(ErrorFlags.None, TimeSpan.FromSeconds(5),
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.